src/routes/api/user/+server.ts

```diff
@@ -15,8 +16,9 @@ export const GET = (async (event) => {
         orgs.map(async (orgId) => {
           const orgInfo = await getOrganizationInfo(orgId);
           return {
-            orgName: orgInfo?.name,
+            orgName: orgInfo?.name ?? null,
             orgId,
+            logoUrl: orgInfo?.logoUrl ?? null,
```

Comment:
Set these to null so they are always present one way or the other, if they are undefined they just drop off the JSON